challenge_13/config.ru

```diff
@@ -0,0 +1,31 @@
+require_relative 'action_dispatch'
+
+class LoggerMiddleware
+  def initialize(app, string)
+    @app = app
+    @string = string
+  end
+
+  def call(environment)
+    puts @string
+    @app.call(environment)
+  end
+end
+
+class CharsetMiddleware
+  def initialize(app, charset: 'UTF-8')
+    @app = app
+    @charset = charset
+  end
+
+  def call(environment)
+    response = Rack::Response[*@app.call(environment)]
+    response.content_type = "#{response.content_type}; charset=#{@charset}"
+    response.finish
+  end
+end
```

Comment:
To state the obvious: you could easily move these pieces of middleware into their own file(s) and `require_relative` them if you wanted to keep `config.ru` small. But since this “configuration” file is literally just Ruby, it’s also fine to have as much code here as you like!